components/user/UserLink.js

```diff
@@ -11,28 +11,12 @@ export default function UserLink({
 }) {
   const [clicks, setClicks] = useState(link.clicks || 0);
   const DisplayIcon = getIcon(link.icon);
-  const clickLink = async () => {
-    try {
-      const res = await fetch(
-        `${BASE_URL}/api/statistics/${username}/${encodeURIComponent(
-          link.url
-        )}`,
-        { method: "PUT" }
-      );
-      const data = await res.json();
-      setClicks(data.clicks);
-    } catch (e) {
-      // TODO: link not found
-      console.log("ERROR link not found ", e);
-    }
-  };
 
   return (
     <a
-      href={link.url}
+      href={`${BASE_URL}/api/handler/links?username=${username}&url=${encodeURIComponent(link.url)}`}
```

Comment:
Should there be quotes here?

Reply:
Hmm, which quotes? there are already backticks